front/src/router/index.ts

```diff
@@ -301,7 +305,7 @@ const routes: Array<RouteConfig> = [
     props: true,
     meta: {
       needAuthentication: true,
-      requiredRoles: [USER_PROFILE.SUPERADMIN, USER_PROFILE.ADMIN, USER_PROFILE.CREATOR],
+      authorizedRoles: [GROUP_ROLE.ADMIN, GROUP_ROLE.OWNER],
```

Comment:
I don't agree, the administration section of the front is available only for `USER_PROFILE.ADMIN` / `USER_PROFILE.SUPERADMIN` users. (maybe a mistake ?)